content/docs/guides/contribute/code-review.en.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Review process"
+linkTitle: "Review process"
+description: "How to give useful feedback"
+weight: 4
+---
+
+The reviewer/maintainer undertakes to carry out the review quickly, and is also responsible for closing _request changes_, check commit history and quickly merge the _pull request_ if allowed.
+
+We propose you a few tips and recommendations that we think are relevant to a human, relevant and rewarding code review for all contributors:
+- [How to Make Your Code Reviewer Fall in Love with You?](https://mtlynch.io/code-review-love/) by Michael Lynch.
```

Comment:
minor (feel free to resolve if you disagree): this link concerns "submit changes" and is already there, so it might be removed here.

Reply:
done!